src/main/java/anichan/parser/AddToWatchlistParser.java

```diff
@@ -62,7 +54,7 @@ private void parameterParser(String fieldGiven) throws AniException {
         isIntegerCheck(fieldValue, ANIME_ID);
 
         try {
-            addToWatchlistCommand.setAnimeIndex(Integer.parseInt(fieldValue));
+            return Integer.parseInt(fieldValue);
         } catch (NumberFormatException e) {
             throw new AniException(OUT_OF_BOUND_INDEX_ERROR);
         }
```

Comment:
Could this parseInt() be replaced with the parseStringToInteger method in CommandParser?

Reply:
You're right. I missed that. Thanks for pointing it out!